app/Http/Controllers/MessagesController.php

```diff
@@ -92,7 +92,9 @@ public function sendMessage(Message $message)
     {
         // Get competition with activity.
         $competitionId = request('competition_id');
-        // $competition = null;
+
+        $contest = Contest::find(request('contest_id'));
+        $this->manager->appendCampaign($contest);
```

Comment:
For example here: https://github.com/DoSomething/gladiator/blob/a924244398fe05108a1009867c04c9f3181f41f4/app/Http/Controllers/ContestsController.php#L86